datafusion-cli/Cargo.toml

```diff
@@ -29,9 +29,9 @@ rust-version = "1.62"
 readme = "README.md"
 
 [dependencies]
-arrow = { version = "20.0.0" }
+arrow = "21.0.0"
 clap = { version = "3", features = ["derive", "cargo"] }
-datafusion = { path = "../datafusion/core", version = "11.0.0" }
```

Comment:
We need to retain these `version` attributes otherwise it will break the release process, IIRC